ResearchKit/Common/ORKTableContainerView.m

```diff
@@ -280,7 +280,7 @@ - (CGSize)keyboardIntersectionSizeFromNotification:(NSNotification *)notificatio
 
 - (void)scrollCellVisible:(UITableViewCell *)cell animated:(BOOL)animated {
     
-    if (cell == nil) {
+    if (cell == nil || [cell respondsToSelector:@selector(bounds)] == NO) {
```

Comment:
If something is calling this method with an object that is not either nil or a UITableViewCell, then that is the bug. Do you have a stack trace for the crash you hit?